### DISCUSSION TITLE
Theme Mimic available for those who want to test it

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3525] [Show & Tell]
I released the source files of the theme I use called MIMIC

As already mentioned on other occasions, the theme is based on RWD and aims to be very similar to the Magento 2 LUMA theme

Over the years I have made small changes to some RWD theme files to be able to display attributes and small thumbnails of the products.

I will write more details over time.

for those who want it, the theme is here:

https://github.com/Danieleeffe1/OpenMage-Themes

there is also a link to view the demo which I also report here

https://teloporto.eu

enjoy it

**Comment by fballiano** [comment]
thank you for sharing this!

**Comment by empiricompany** [comment]
this is a good starting point @Danieleeffe1! I am also studying how to develop a new version of RWD

IMHO it make sense to renew the rwd theme if we can accomplish at least this tasks:

- use flexbox and grid css instead of float 
- migrate all javascript code (for cart popover for example) from jquery to vanillajs / typescript or some light framework 
- make webp format for images by default
- remove prototype.js: at least start to remove in some areas except for the checkout which is more complex like @fballiano done with enquire https://github.com/OpenMage/magento-lts/pull/3208 or here for example https://github.com/OpenMage/magento-lts/pull/3230. 
I think to form validation that can be done in native HTML, swatches but is a bit challenging etc...

when we have at least first 2 points, it can make the difference and could be a good base for new child themes!

**Reply by Danieleeffe1** [reply]
Thank you

My knowledge is very basic

I don't know javascript, much less vanilla or wnindows.media

I am a pure and simple designer (I viscerally repudiate any form of code, it drives me haywire).

I have some knowledge of CSS and a lot of patience to search the web for the solutions I need.

- As regards css Flexbox and float I'm slowly progressing

- for the conversion to webp I install the free yireo module

- for the minicart I found many plugins developed for Magento 1, one in particular could be interesting to take up again but it is very old and has problems.

all the minikart plugins that I have found have the same problem with configurable products: when you enter the quantity and purchase it is doubled in the checkout.

Over time I have found and archived a huge amount of free modules for Magento 1

I will try to be as helpful as possible, but within my limits.

**Comment by Danieleeffe1** [comment]
I updated the project, now the mobile menu appears with css animations

the result can be seen 

https://teloporto.eu

**Comment by kiatng** [comment]
Nice looking theme. So I did a quick test, my feedback:
1. On [product page](https://teloporto.eu/black-nolita-cami.html), the validation of required options is server side, and requires a page reload.
2. On the same product page, I selected color Pink and click the _Add to Cart_ button, the page losses the earlier Pink selection. It's better UI to remember the customer previous selection. 

[edit] Item 2 is the default core behavior. I fixed this long long time ago. I will make a PR soon for it.

**Reply by Danieleeffe1** [reply]
Thanks for your checking.

the system is openmage 20.1.1
php 7.4
apache 2.4
magento 1.9.0 sample data
Redis enabled
Lesti FPC enabled

I worked only and essentially on the CSS and modified the HTML of the pages by adding divs to obtain different aspects.

I don't know how to solve the problems you mentioned

can you show me some tutorials?
